### PR TITLE
Add "json" as supported format for `lingui-translation-status`

### PR DESCRIPTION
**Pull request by igorDolzh**
and add more languages

https://linear.app/pleo/issue/INT-1059/add-translation-status-for-mail-templates

### CODE REVIEW

File: lingui-translation-status/lib/main.js [hunk 1]
```diff
@@ -61,6 +71,9 @@ function getPattern(format) {
     if (format === 'po') {
         return /\+msgid "(.*)".*\n\+msgstr.""|msgid "(.*)".*\n\-msgstr ".*".*\n\+msgstr.""/;
     }
+    if (format === 'json') {
+        return /\+.*"(.*)":.*""/;
```
Comment (bel0v) [inline lingui-translation-status/lib/main.js:75]: https://i.kym-cdn.com/photos/images/original/000/719/435/49b.gif

**Review by bel0v — APPROVED** [review]

File: lingui-translation-status/lib/main.js (hunk 1 above)
Comment (dariye) [reply lingui-translation-status/lib/main.js:75]: AH I get get it. It's how it extracts the key:value from the json format

**Review by dariye — APPROVED** [review]
Nice!; Is this for mobile? It's the only project that uses json format for translations afaik

**Comment by igorDolzh** [comment]
> Nice!; Is this for mobile? It's the only project that uses json format for translations afaik

It could be used for mobile, right, and also for mail-templates

**Pull request merged**